content/blog/2022/10/2022-10-20-jenkins-election-announcement.adoc

```diff
@@ -0,0 +1,97 @@
+---
+:layout: post
+:title: "Jenkins Board and Officer Elections 2022"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- kmartens27
+description: >
+    Nomination and voter registration period 2022 now open for Jenkins Board elections.
+links:
+  discourse: true
+opengraph:
+  image: /images/governance/elections/2022/2022-opengraph.png
+---
+
+We are excited to announce the 2022 Jenkins Governance Board elections!
+The nomination and voter registration period is now open.
+Nominations can be submitted for two link:/project/board[governance board] positions, and all five link:/project/team-leads/[officer] positions, namely: Security, Events, Release, Infrastructure, and Documentation.
+
+Thank you to link:https://github.com/halkeye[Gavin Mogan], link:https://github.com/ewelinawilkosz[Ewelina Wilkosz], link:https://github.com/dduportal[Damien Duportal], link:https://github.com/wadeck[Wadeck Follonier], link:https://github.com/alyssat[Alyssa Tong], link:https://github.com/timja[Tim Jacomb], and link:https://github.com/MarkEWaite[Mark Waite] for all of your work over the last term. 
+
+These roles are crucial to the expanding community and continued progress of Jenkins.
+This is an opportunity for Jenkins contributors and community members to sign-up to vote in elections, and acknowledge other community members by nominating them for a position.
+The deadline for nominations and voter registration is *November 9th*.
+
+Participation in the election process requires an account on link:https://community.jenkins.io[community.jenkins.io].
+You can use your Github account or create a new discourse account for link:https://community.jenkins.io[community.jenkins.io].
+Additionally, you must be able to showcase at least one contribution done before the September 1st, 2022.
+As mentioned on link:/participate/[jenkins.io/participate], they are many different ways to contribute to Jenkins despite having different scopes.
+This makes it difficult to consistently measure contributions.
+Therefore, we are trusting participants and not requiring that you provide evidence of contribution as part of your voter registration.
+We reserve the right to ban the specific account from the election process if we identify abuse.
+
+The election has 3 stages:
+
+* Nomination and voter registration
+* Voting
+* Results announcement
```

Comment:
Could the deadline be added here as well or would it be too much?

Reply:
Yup, I don't think it's too much, and it's probably best to highlight the important info such as dates/time frames. I've added them in, thanks for the suggestion @alecharp